pygments/styles/friendly_grayscale.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+"""
+    pygments.styles.friendly_grayscale
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    A style based on friendly style.
+    The color values of the friendly style have been converted to grayscale using the luminosity value calculated by http://www.workwithcolor.com/color-converter-01.htm
+
+    :copyright: Copyright 2006-2017 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+
+    ! Name.Attribute is set to black because of https://github.com/gpoore/minted/issues/103
```

Comment:
shouldn't we fix that bug instead?

Reply:
Currently, I don't feel empowered to fix that issue. I use the real value and put on my todo list to open an issue here (extracting the "best of" of https://github.com/gpoore/minted/issues/103).